File: core Java/Assignment/Function Overloading/Time.java
class Time
{
 int hr , min , sec;

void add ( int h , int m , int s)
{
hr= hr+h;
min= min+m;
sec = sec+s;

if(sec>=60)
{
sec = sec - 60 ;
min = min + 1 ;
}

if ( min >= 60)
{
min = min - 60 ;
hr = hr + 1;
}

if( hr >= 24)
{
hr= hr  - 24 ;
}
System.out.println( " time is " +hr + ":" + min+ ":" + sec);
 }
}
class TimeTest
{
public static void main(String [] args)
{
 int hr= 25 , min=11 , sec = 50;

Time t1;
t1= new Time();
t1.add( hr , min , sec);


Time t2;
t2= new Time();
t2.add( hr , min , sec);

 }
} 




 

File: C Programming/Assignment/Assignment 14/Time.c
#include<stdio.h>
#include<string.h>
struct Time
{
	int Hour;
	int Min;
	int sec;
};
void main()
{
	struct Time T1,T2;
	T1.Hour=2;
	T1.Min=30;
	T1.sec=15;
	printf("%d %d %d",T1.Hour,T1.Min,T1.sec);
	
	printf("enter the details of time 2\n");
	scanf("%d",&T2.Hour);
	scanf("%d",&T2.Min);
	scanf("%d",&T2.sec);
	printf("Hour=%d Min=%d sec=%d",T2.Hour,T2.Min,T2.sec);
}